GoogleApiWrappers/src/Google.Apis.Storage.v1.ClientWrapper/DownloadObjectOptions.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright 2015 Google Inc. All Rights Reserved.
+// Licensed under the Apache License Version 2.0.
+
+using Google.Apis.Download;
+
+namespace Google.Apis.Storage.v1.ClientWrapper
+{
+    /// <summary>
+    /// Options for <c>DownloadObject</c> operations.
+    /// </summary>
+    public class DownloadObjectOptions
+    {
+        // TODO: Retry policy? There's not an awful lot else to specify.
```

Comment:
Hmm.. We're not actually using that API, but the media downloader - which I don't *think* supports the same options.

I believe we can specify a generation though. We might want to only use the generation from the Object source though, rather than having it in options. (I doubt that you'd have the generation but not the Object itself.)

Will see if I can find out whether the media API allows the "if" conditions... and will try to work out how we'd signal that to the caller...